static/js/controllers/contacts-content.js

```diff
@@ -11,11 +11,12 @@
       $scope.selectContact($stateParams.id);
 
       $scope.$on('ContactUpdated', function(e, contact) {
-        if (!contact || (
-            contact._deleted &&
-            $scope.selected &&
-            $scope.selected._id === contact._id)) {
+        if (!contact) {
```

Comment:
Seems fine.  I wouldn't add the list of contacts until we actually see a need for that though.
I guess it's the event's name which misled me - I guess it's more literally `Contact(s)Updated`.  Not suggesting we change it, though!